doc/source/examples/multiphysics/3d-dam-break/3d-dam-break.rst

```diff
@@ -194,7 +194,7 @@ In the ``source term`` subsection, we define the gravitational acceleration.
 VOF
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In the ``VOF`` subsection, we enable ``interface sharpening`` to reconstruct the interface and keep it sharp during the simulation. Here we use the default ``constant`` method for interface sharpening.
+In the ``VOF`` subsection, we select the ``tanh`` filter to filter the phase fraction and get a more defined interface. We set the value of beta to 10.
```

Comment:
the default value for beta is 20, so maybe we should leave it at that for this example.

Reply:
I tried running for beta=20 on Narval, but it failed .-.